src/app/ui/components/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Tabs } from './tabs';
+
+const meta: Meta<typeof Tabs.Root> = {
+  component: Tabs.Root,
+  tags: ['autodocs'],
+  title: 'Tabs',
+};
+
+export default meta;
+type Story = StoryObj<typeof Tabs.Root>;
+
+export const Primary: Story = {
```

Comment:
I did use the pattern @pete-watters suggested on importing `Tabs as Component` so the basic story component can be named `Tabs` rather than `Primary`, but I am pretty open to whichever people feel is best? We can change the two I merged if needed.

Reply:
I like doing that as then you don't have a huge amount to select from in the UI. I guess it depends. 

In their demo video you can see they have `Primary`, `Secondary`, `Large`, `Small` buttons:
https://storybook.js.org/d61da811850135f33ca417b1b3e6fa92/example-browse-all-stories-optimized.mp4 

But you can also just change the variant using the `Controls` in the UI so it makes more sense to me to have the stories names per component then you can change variants with controlled.

It also lends us to refactoring to a flatter structure if we want, so we can group a lot of stories together into one expandable list. 

It could be something to feel out as we add components. 

For some things I don't think it makes sense to say `Primary` if there is no secondary, it just adds confusion

Reply:
Happy to call it `Component`. Though note this only requires exporting as the name "Component" and not renaming all variables within the story, as is done in the select story.

Reply:
I don't understand what you mean here? When storybook is run, the export has to be named what you want to see in the UI?